sdk/eventhub/azure-eventhubs/azure/eventhub/eventprocessor/__init__.py

```diff
@@ -0,0 +1,17 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# -----------------------------------------------------------------------------------
+
+from .event_processor import EventProcessor
+from .partition_processor import PartitionProcessor, CloseReason
+from .partition_manager import PartitionManager
+from .sqlite3_partition_manager import Sqlite3PartitionManager
+
+__all__ = [
+    'CloseReason',
+    'EventProcessor',
+    'PartitionProcessor',
+    'PartitionManager',
+    'Sqlite3PartitionManager',
```

Comment:
We should not include a Sqlite based partition manager in the main namespace. I would assume that one of the main reasons to use eph is to spread out load over multiple machines. Which means that sqlite is not the right answer. I would even consider ripping this out and make it into a separate package for now.

Reply:
We decided to include a sample PartitionManager in the main package at least in preview2 for users to try out. Or else they'll have to make their own PartitionManager to try on other APIs. Other languages have an in-memory PartitionManager in the main package. Python happens to have Sqlite in the standard library so I made this one.
For furural release, we may remove this one. Subject to decision by then.